Foundation/URLCache.swift

```diff
@@ -162,7 +191,16 @@ open class URLCache : NSObject {
         @result an initialized URLCache, with the given capacity, backed
         by disk.
     */
-    public init(memoryCapacity: Int, diskCapacity: Int, diskPath path: String?) { NSUnimplemented() }
+    public init(memoryCapacity: Int, diskCapacity: Int, diskPath path: String?) {
+        self.memoryCapacity = memoryCapacity
+        self.diskCapacity = diskCapacity
+
+        if let _path = path {
```

Comment:
You might also want to check that `diskCapacity` is more than zero.

Reply:
Darwin indeed does accept 0 arguments. My assumption is that memory capacity 0 means that as much as possible no data is kept in memory, and disk capacity 0 that no data is written to disk.